activation/handler_v1.go

```diff
@@ -450,19 +445,23 @@ func (h *HandlerV1) checkMalicious(
 	ctx context.Context,
 	tx sql.Transaction,
 	watx *wire.ActivationTxV1,
-) (*mwire.MalfeasanceProof, error) {
+) (bool, *mwire.MalfeasanceProof, error) {
 	malicious, err := identities.IsMalicious(tx, watx.SmesherID)
 	if err != nil {
-		return nil, fmt.Errorf("checking if node is malicious: %w", err)
+		return false, nil, fmt.Errorf("checking if node is malicious: %w", err)
 	}
 	if malicious {
-		return nil, nil
+		return true, nil, nil
 	}
 	proof, err := h.checkDoublePublish(ctx, tx, watx)
 	if proof != nil || err != nil {
-		return proof, err
+		return true, proof, err
 	}
```

Comment:
This is wrong I believe. If `err != nil` I think it should return `false, nil, err`.

Reply:
fixed